src/main/java/com/vinyl/service/UserPasswordValidator.java

```diff
@@ -0,0 +1,24 @@
+package com.vinyl.service;
```

Comment:
we now have a lot of classes that do validation, create a subpackage com.vinyl.service.validation;

Reply:
done